### DISCUSSION TITLE
The console class for some reason became a ambiguous reference for some reason any advice on how to fix this issue.

### DISCUSSION THREAD

**Discussion by SuperWasTaken** [CosmosOS/Cosmos#2777] [Q&A] [answered]
Hey so I got this error randomly while working on my Operating System. It just randomly showed up and I want to know what this error is and how can I fix it here is the full error:

 Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0104	'Console' is an ambiguous reference between 'System.Console' and 'Cosmos.System.Console'	OsNameHere	C:\Users\user\source\repos\PerfexiOS\PerfexiOS\Commands\CommandManger.cs	39	Active

**Comment by 9xbt** [comment, marked as answer]
instead of
```cs
using Cosmos.System;
```
do
```cs
using Sys = Cosmos.System;
```
because cosmos also has a class named `Console` in the `Cosmos.System` namespace. I think vs automatically added that line for you without you realizing which explains why its so random, VS moment:tm:

**Reply by SuperWasTaken** [reply]
Thanks :)
